gavel/templates/vote.html

```diff
@@ -28,11 +28,31 @@ <h1>Current</h1>
     <h1>Vote</h1>
   </div>
   <div class="info">
+
+    <script type="application/javascript">
+      function buttonSelected() {
+          $("#vote-done").prop("disabled", false);
```

Comment:
I'd rather not pull in a large dependency (JQuery) if it's not necessary. Can this be replaced with something like `document.getElementById('vote-done').disabled = false`?

Reply:
Also, this is the only use of JavaScript in this project, I somehow managed to avoid it otherwise. It would be kinda nice if we could somehow get this functionality without JS, but I don't know if that's easily possible.

Would it be good to have some sort of `<noscript>` somewhere that warns the user if they have JavaScript disabled?